nepalingo-web/src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+import useDictionary, { DictionaryProps, DictionaryResponse } from '../../hooks/useDictionary';
+
+const SearchBar: React.FC = () => {
```

Comment:
Let's rename this to be more specific - maybe `DictionarySearchBar`? and `DictionaryPage`.

SearchBar could be for any kind of data